libs/core/src/lib/avatar-group/directives/avatar-group-overflow-body.directive.ts

```diff
@@ -22,4 +39,84 @@ export class AvatarGroupOverflowBodyDirective {
     /** Hide vertical scrollbar from the overflow body. */
     @Input()
     noVerticalScroll = true;
+
+    /** @hidden Avatar Group Overflow items. */
+    @ContentChildren(forwardRef(() => AvatarGroupFocusableAvatarDirective), { descendants: true })
+    overflowItems: QueryList<AvatarGroupFocusableAvatarDirective>;
+
+    /** @hidden FocusKeyManager instance */
+    private _keyboardEventsManager: FocusKeyManager<AvatarGroupFocusableAvatarDirective>;
+
+    /** @hidden */
+    private readonly _subscription = new Subscription();
+
+    /** @hidden handles rtl service */
+    private _dir: 'ltr' | 'rtl' | null = 'ltr';
+
+    /** @hidden */
+    constructor(@Optional() private _rtlService: RtlService) {}
+
+    /** @hidden */
+    ngAfterViewInit(): void {
+        this._listenForItemChanges();
+
+        this._setKeyboardEventsManager();
+
+        this._subscribeToRtl();
+    }
+
+    /** @hidden */
+    ngOnDestroy(): void {
+        this._subscription.unsubscribe();
+    }
+
+    /** @hidden */
+    @HostListener('keyup', ['$event'])
+    keyUpHandler(event: KeyboardEvent): void {
+        if (KeyUtil.isKeyCode(event, [TAB])) {
+            const index = this.overflowItems.toArray().findIndex((item) => item._element === event.target);
+            if (index !== -1) {
+                this._keyboardEventsManager.setActiveItem(index);
+            }
+        }
+
+        if (KeyUtil.isKeyCode(event, [DOWN_ARROW, UP_ARROW, LEFT_ARROW, RIGHT_ARROW])) {
+            // passing the event to key manager so we get a change fired
```

Comment:
We need to preventDefault on these arrow key events to prevent window scrolls

Reply:
hi @mikerodonnell89 
onKeydown provide preventDefault itself
https://github.com/angular/components/blob/master/src/cdk/a11y/key-manager/list-key-manager.ts#L300